README.md

```diff
@@ -15,7 +15,7 @@
 ![screenshot](./screenshots/screen2.png)
 Covid Tracker is site used to check covid updates.
 ## [Live Demo](https://ncov-site.netlify.app/)
-## [Presentation](#)
+## [Presentation](https://youtu.be/YaC8EksPYMc)
```

Comment:
Please ensure that the app has integration tests using React Testing Library according to the project requirement.

* You can perform a mocking data fetching and Redux Store connection.